src/bptl/work_units/zgw/tasks/zaak_relations.py

```diff
@@ -299,3 +299,53 @@ def perform(self) -> dict:
         )
 
         return {}
+
+
+@register
+class CreateZaakObject(ZGWWorkUnit):
+    """
+    Create a new ZAAKOBJECT for the ZAAK in the process.
+
+    **Required process variables**
+
+    * ``zaakUrl``: full URL of the ZAAK to create a new ZaakObject for
+    * ``objectUrl``: full URL of the OBJECT to set
+    * ``objectType``: type of the OBJECT
+    * ``services``: JSON Object of connection details for ZGW services:
+
+        .. code-block:: json
+
+          {
+              "<zrc alias>": {"jwt": "Bearer <JWT value>"},
+          }
+
+    **Optional process variables**
+
+    * ``objectTypeOverige``: description of the OBJECT type if objectType = 'overige'
+    * ``relatieomschrijving``: description of relationship between ZAAK and OBJECT
+
+    **Optional process variables (Camunda exclusive)**
+
+    * ``callbackUrl``: send an empty POST request to this URL to signal completion
+
+    **Sets the process variables**
+
+    * ``zaakobject``: the full URL of the created ZAAKOBJECT
+    """
+
+    def create_zaakobject(self) -> dict:
+        variables = self.task.get_variables()
+        zrc_client = self.get_client(APITypes.zrc)
+        data = {
+            "zaak": variables["zaakUrl"],
```

Comment:
can you use the `check_variable` function, as it provides better error feedback when variables are missing

Reply:
check_variable is used for required variables